fire/SFMainMod.F90

```diff
@@ -758,41 +756,38 @@ subroutine  area_burnt ( currentSite )
              
              ! INTERF-TODO:
```

Comment:
I think we should remove these two comments, I think I wrote them and they don't seem relevant anymore.